test/test_mpc.py

```diff
@@ -241,11 +251,12 @@ def test_arithmetic(self):
                 encrypted_out = getattr(encrypted1, func)(encrypted2)
                 self._check(encrypted_out, reference, "private %s failed" % func)
 
-            tensor = self._get_random_test_tensor(is_float=True)
-            reference = tensor * tensor
-            encrypted = MPCTensor(tensor)
-            encrypted_out = encrypted.square()
-            self._check(encrypted_out, reference, "square failed")
+        # test square
+        tensor = self._get_random_test_tensor(is_float=True)
+        reference = tensor * tensor
+        encrypted = MPCTensor(tensor)
+        encrypted_out = encrypted.square()
+        self._check(encrypted_out, reference, "square failed")
 
         # Test radd, rsub, and rmul
```

Comment:
The rest of this test has tests for "right" operations like `__radd__`, __rsub__`, and `__rmul__`.